### DISCUSSION TITLE
Uploading my Profile along with my pictures

### DISCUSSION THREAD

**Discussion by Bencode2233** [community/community#142167] [New to GitHub]
Good Morning , I did built my index html (css, js) and create a folder that contain the code index.html and a picture.
when I started to create new repository and upload the folder. and when I hit the link ill comes with error 404.
and I tested without picture the link works and open my profile .
Please do help me how to have this work 
![main](https://github.com/user-attachments/assets/626a0007-2550-47bf-94d4-82d761c88d9b)
![swift](https://github.com/user-attachments/assets/af645697-777b-42b0-af01-f7de20e1e6c5)

**Comment by luannndev** [comment]
Hi Bencode2233,

Good morning! It sounds like you're having trouble uploading your profile and images to your GitHub repository. A 404 error typically means that the server cannot find the requested resource. Here are some steps to help resolve the issue:

Check Repository Structure: Make sure that the index.html file and the folder containing your images are properly structured within the repository. The index.html should be in the root directory of your repository if you want it to be the main page.

Ensure File Names Are Correct: Check the file names of your images in the index.html file. Ensure that the paths to the images are correct and match the folder structure in your repository.

Linking Images in HTML: Use relative paths to link to your images in the HTML code. For example, if your images are in a folder named images, your HTML should reference them like this: <img src="images/your-image.jpg" alt="Description">.

Commit and Push Changes: After making changes to your files, ensure that you commit and push them to the repository. Use the following commands:

git add .
git commit -m "Upload index.html and images"
git push origin main

Check GitHub Pages Settings: If you are using GitHub Pages to host your site, make sure that GitHub Pages is enabled for your repository in the repository settings. Choose the correct branch (usually main) for publishing.

Wait a Moment: After making these changes, give GitHub a moment to process them before you try accessing the link again.

If you follow these steps and still encounter issues, feel free to provide more details about your repository structure, and I'll be happy to help further!

**Reply by Bencode2233** [reply]
hi hanks for the infos , yes I did enter the picture in the index.html code as relative path , tried all method but really nothing helps .
the only way I can host on the free public repository is a profile without a picture !
and in fact I did check the image was loaded ,and was all all structured files in one folder also did test the code and working properly on any browser except GitHub.Get really frustrated ! thanks again

**Comment by zshaian** [comment]
I visited your repository, but the link is showing a 404 error page when clicked. I noticed that you uploaded a folder named `file`, which seems to be causing the issue. I would recommend extracting the contents of the `file` folder, as it is unnecessary to keep them inside.

i also noticed, you've placed your `index.html` file inside a folder also named `index.html`, which is contributing to the problem. You should extract the index.html file from that folder and then remove the extra index.html folder.

Your file and folder structure should look something like this:
**Example 1**

jet/
│
├── index.html
│
└── images/
    └── main.png
    
**Example 2**

![example image](https://i.imgur.com/pHaL0FE.jpeg)

---
i'm referring to this repository. Let me know if this is not the repo the you are having trouble with: [repo link](https://github.com/Bencode2233/jet)

P.S: Also, ensure that the file extension for your image is in lowercase. Instead of "PNG," it should be "png."

**Comment by Bencode2233** [comment]
Thanks a lot a much appreciated , I did try all methods ,but will try this one .In fact I'm trying to post my profile as developer along with my picture and other pictures. its like a website but in single page so I can attached to my resume as freelancer .
thanks again
